docs-api/frontity-packages/collections-packages/hooks/infinite-scroll-hooks.md

```diff
@@ -277,6 +222,8 @@ const PostType = () => {
 export default connect(PostType);
 ````
 
```

Comment:
Does the `<PostTypeEntity>` component imported and used in the code above know what to do to render the post/page content? Can it do what it needs to do based on just the link prop passed to it? 

If not it seems to me that more functionality is needed in the `posts.map` function in order to render the post content.

Reply:
I guess using this component `PostTypeEntity` It's a way of saying: this component will manage the logic needed to display the content of a specific post type for this route

I wouldn't add further explanations here.

Reply:
That's what I guessed, but wondered if the link prop is enough information for it to work with.